### ISSUE TITLE
Adding a column shouldn't trigger a `adding-foreign-key-constraint` violation

### ISSUE THREAD

**Issue by reteps** [sbdchd/squawk#647]
```sql
ALTER TABLE grading_jobs
ADD COLUMN deleted_by BIGINT REFERENCES users (user_id) 
ON UPDATE CASCADE ON DELETE CASCADE;
```

- Since the column was just added, there isn't any columns that need to be validated, and this would succeed instantly.
